_source/_includes/navigation.html

```diff
@@ -2,6 +2,8 @@
   <ul>
     <!-- <li class="{% if page.class == 'speakers' %}active{% endif %}"><a href="/speakers">Speakers</a></li> -->
     <!-- <li class="{% if page.class == 'schedule' %}active{% endif %}"><a href="/schedule">Schedule</a></li> -->
+    <li class="{% if page.class == 'cpf' %}active{% endif %}"><a
```

Comment:
yes it should be `cfp`, check provided a new token, record a new cassette, change the code to match `pullreviewtester` user